src/core/tests/frontend/paddle/test_models/gen_scripts/generate_tile.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# tile paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle
+import sys
+from paddle.fluid.layers.tensor import fill_constant
+
+
+def paddle_tile(name: str, x, repeat_times, to_tensor=False, tensor_list=False):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(
+            name="x",
+            shape=x.shape,
+            dtype=x.dtype if x.dtype != np.bool_ else np.int32,
+        )
+        node_x = paddle.cast(node_x, dtype=x.dtype)
+        repeat_times_list = []
+        if tensor_list:
+            for i in repeat_times:
+                temp_out = fill_constant([1], "int32", i, force_cpu=True)
+                repeat_times_list.append(temp_out)
+        else:
+            repeat_times_list = repeat_times
+
+        if to_tensor:
+            repeat_times_list = paddle.static.data(
+                name="repeat_times",
+                shape=repeat_times.shape,
+                dtype=repeat_times.dtype,
+            )
+        out = paddle.tile(node_x, repeat_times_list)
+
+        if out.dtype == paddle.bool:
+            out = paddle.cast(out, dtype=np.int32)
+
+        cpu = paddle.static.cpu_places(1)
+        exe = paddle.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(paddle.static.default_startup_program())
+
+        if x.dtype == np.bool_:
+            x = x.astype(np.int32)
+
+        if to_tensor:
+            feed = {"x": x, "repeat_times": repeat_times}
+        else:
+            feed = {"x": x}
+        outs = exe.run(feed=feed, fetch_list=[out])
+
+        saveModel(
+            name,
+            exe,
+            feedkeys=[*feed.keys()],
+            fetchlist=[out],
+            inputs=[*feed.values()],
+            outputs=[outs[0]],
+            target_dir=sys.argv[1],
+        )
+    return outs[0]
+
+
+def main():
+    test_cases = [
+        "float32",
+        "int32",
+        "int64",
+        "bool",
+    ]
+    for case in test_cases:
+        x = np.array([1, 2, 3]).astype(case)
+        paddle_tile("tile_list_" + case, x, [2, 1])
+        paddle_tile("tile_tuple_" + case, x, (2, 2))
+
+    x = np.array([1, 2, 1]).astype("float32")
+    repeat_times = np.array([2, 1]).astype("int32")
+    paddle_tile("tile_tensor_list", x, repeat_times, tensor_list=True)
+    paddle_tile("tile_repeat_times_tensor", x, repeat_times, to_tensor=True)
```

Comment:
The input X could be of more than 1 dimensions. And there could be cases like "Example 2: number of elements in “repeats” is more than shape of “data”" referring to https://docs.openvino.ai/latest/openvino_docs_ops_movement_Tile_1.html.

Could you please use a higher dimension (like 2 or 3 which is prone to have problems) instead of 1 and cover the case I mentioned?

Reply:
Thanks for contribution. Nice PR and close to be qualified.

Reply:
I have added unit tests for dimension inconsistency.

![image](https://user-images.githubusercontent.com/27223114/183552415-adab7105-d62f-43f6-b12e-060c21b92b3c.png)

thanks for your review